{{ .ProjectSnake }}/gazelle_python.yaml

```diff
@@ -0,0 +1,57 @@
+# GENERATED FILE - DO NOT EDIT!
+#
+# To update this file, run:
+#   bazel run //:gazelle_python_manifest.update
+
+manifest:
+  modules_mapping:
+    google.protobuf: protobuf
```

Comment:
It comes with protobuf?

Reply:
Yes, when I added the third-party dep support I hit a problem: pip-compile refuses to compile zero dependencies. So the CI here can't be green unless the project has *some* dep.

`protobuf` is chosen here a bit out of laziness: I don't know how to make the `requirements-lock.txt` contain templating markup, and so it's probably tricky to make that dep only appear if you chose protobuf included in the scaffold form.

WDYT?

Reply:
Yeah that's fine, protobuf is as good as any. An alternative would be to add some toy dependency like cowsay but then you have to reason about users having to remove it.